distribution/test/itests/pom.xml

```diff
@@ -78,53 +78,56 @@
         </dependencies>
     </dependencyManagement>
     <dependencies>
+        <!--&lt;!&ndash; Start of dependencies for pax exam karaf container &ndash;&gt;-->
         <dependency>
-            <groupId>org.codice.ddf</groupId>
-            <artifactId>ddf</artifactId>
-            <version>${project.version}</version>
-            <type>zip</type>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.googlecode.pojosr</groupId>
-                    <artifactId>de.kalpatec.pojosr.framework</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>ch.qos.logback</groupId>
-                    <artifactId>logback-classic</artifactId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.ops4j.pax.exam</groupId>
+            <artifactId>pax-exam-junit4</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
-            <version>1.7.7</version>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
         </dependency>
+        <!-- End of pax exam dependencies -->
         <dependency>
-            <groupId>ddf.test.thirdparty</groupId>
-            <artifactId>rest-assured</artifactId>
+            <groupId>org.apache.karaf.features</groupId>
+            <artifactId>org.apache.karaf.features.core</artifactId>
+            <version>${karaf.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.ops4j.pax.tinybundles</groupId>
-            <artifactId>tinybundles</artifactId>
+            <groupId>org.apache.karaf.shell</groupId>
+            <artifactId>org.apache.karaf.shell.core</artifactId>
+            <version>${karaf.version}</version>
         </dependency>
-        <!-- Start of dependencies for pax exam karaf container -->
         <dependency>
-            <groupId>org.ops4j.pax.exam</groupId>
-            <artifactId>pax-exam-junit4</artifactId>
+            <groupId>ddf.admin.core</groupId>
+            <artifactId>admin-core-appservice</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.ops4j.pax.exam</groupId>
-            <artifactId>pax-exam</artifactId>
+            <groupId>org.springframework.osgi</groupId>
+            <artifactId>spring-osgi-core</artifactId>
+            <version>1.2.1</version>
```

Comment:
Could we use properties for this (as well as for `org.apache.felix.configadmin` and `rest-assured`) instead of hard-coding the versions?